package.json

```diff
@@ -185,7 +185,6 @@
     "open": "0.0.5",
     "prettydiff2": "^2.2.7",
     "pug-beautify": "^0.1.1",
-    "remark": "6.0.1",
```

Comment:
@kachkaev @szeck87 : I think we can support both Remark-CLI and be backwards compatible with `remark` as a Node.js dependency.

Since the `Remark CLI` is now an Executable we should be able to determine if it is installed or not.
See https://github.com/Glavin001/atom-beautify/blob/30dece209a50868c5ce8589b69a79240e91e5ea4/src/beautifiers/executable.coffee#L99-L107

I'm thinking something along the lines of:

```coffeescript
isCliInstalled = @exe("remark").isInstalled

if isCliInstalled
  # ...
  @exe("remark").run(remarkArgs...
  # ...
else
  # ...
  cleanMarkdown = remark().process(text, options).toString()	
  # ...
```

I would prefer to add new features while being backwards compatible. Removable of `remark` as a builtin dependency is considered a breaking change to me. However, there is value in using `Remark CLI` directly 👍, such as the configuration file.